### DISCUSSION TITLE
Plans for support of the rest of the attiny chips

### DISCUSSION THREAD

**Discussion by Sandvoxel** [Rahix/avr-hal#518] [Q&A]
I am currently working with a project with a attiny 3227 but the lack of I2C is kind of making me back away. I was just wondering what the roadmap was for that as it seems that the attiny mcus have gone kind of stagnant for the last bit.

**Comment by Rahix** [comment]
Support for MCUs is added as needed.  So if you are interested in ATtiny3227, I encourage you to add support for this chip in `avr-device` and `avr-hal` :) If you need help along the way, don't be afraid to ask any and all questions you might have!  If you need some pointers to get started, the README in `avr-device` outlines the first steps.
